jenkins/build.sh

```diff
@@ -68,19 +113,24 @@ INSTALL_PATH="/opt/seagate/""${PRODUCT}"
 
 mkdir -p "$INSTALL_PATH"
 
+# build addb plugin
+if [ "$BUILD_ADDB" == "true" ]; then
+  echo "building addb plugin and adding this binary file as a part of cortx-rgw-integration rpm."
```

Comment:
instead if 'this binary file', should we name the file?

Reply:
yes. added the name of binary file.